demisto_sdk/commands/lint/tests/test_pylint_plugin/base_checker_test.py

```diff
@@ -739,3 +739,143 @@ def test_indicators_doesnt_exist(self):
+        ):
+            self.checker.visit_call(node_b)
+
+    def test_execute_command_with_set_incident_and_id_exists(self):
+        """
+        Given:
+            - String of a code part which is being examined by pylint plugin.
+        When:
+            - executeCommand function exists in the code with the 'setIncident' build in command
+              and a given incident id.
+        Then:
+            - Ensure that the correct message id is being added to the message errors of pylint
+        """
+        self.checker.is_script = True
+        _, node_b = astroid.extract_node("""
+            def test_function(): #@
+                demisto.executeCommand('setIncident', {'id': 'incident_id', 'name': 'incident_name'}) #@
+        """)
+        assert node_b is not None
+        with self.assertAddsMessages(
+                pylint.testutils.Message(
+                    msg_id='missing-permission',
+                    node=node_b,
+                )
+        ):
+            self.checker.visit_call(node_b)
+
+    def test_execute_command_with_set_incident_and_id_in_args_exists(self):
+        """
+        Given:
+            - String of a code part which is being examined by pylint plugin.
+        When:
+            - executeCommand function exists in the code with the 'setIncident' build in command
+              and a given incident id which is predefined in 'args'.
+        Then:
+            - Ensure that the correct message id is being added to the message errors of pylint
+        """
+        self.checker.is_script = True
+        _, node_b, node_c = astroid.extract_node("""
+            def test_function(): #@
+                args = {'name': 'incident_name', 'id': 'incident_id'} #@
+                demisto.executeCommand('setIncident', args) #@
+        """)
+        assert node_b is not None and node_c is not None
+        with self.assertAddsMessages(
+                pylint.testutils.Message(
+                    msg_id='missing-permission',
+                    node=node_c,
+                )
+        ):
+            self.checker.visit_call(node_c)
+
+    def test_execute_command_with_set_incident_and_without_id_in_args_exists(self):
+        """
+        Given:
+            - String of a code part which is being examined by pylint plugin.
+        When:
+            - executeCommand function exists in the code with the 'setIncident' build in command without incident id.
+        Then:
+            - Ensure that there is no errors, Check that there is no error message.
+        """
+        self.checker.is_script = True
+        _, node_b, node_c = astroid.extract_node("""
+            def test_function(): #@
+                args = {'name': 'incident_name'} #@
+                demisto.executeCommand('setIncident', args) #@
+        """)
+        assert node_b is not None and node_c is not None
+        with self.assertNoMessages():
+            self.checker.visit_call(node_b)
+            self.checker.visit_call(node_c)
```

Comment:
let's add a test to see this test is skipped if runas is set.

Reply:
done.